src/picknik_ur_base_config/description/picknik_ur.xacro

```diff
@@ -98,14 +98,6 @@
     <origin xyz="0.0 0.0 0.134" rpy="0.0 0.0 ${pi}" />
   </joint>
 
-  <!-- Moveit Studio requires a link name called manual_grasp_link to perform quick tasks such as inspect surface -->
```

Comment:
Should we preserve this comment for `grasp_link` in all configs?

Reply:
The comment is still in the `gripper_and_camera.xacro`, `robotiq_2f_85_gripper.urdf`, and `config.yaml`, so I'm OK with removing this whole block.

To me, it doesn't make much sense to have a standalone comment where there no longer is a `manual_grasp_link` or `grasp_link` definition.

What do you think?

Reply:
Grasp link is defined on line 94 and this URDF is calling
`<xacro:include filename="$(find robotiq_description)/urdf/robotiq_2f_85_macro.urdf.xacro" />` not the one in picknik_accessories. FYI if you want to use PickNik's xacro we need the ability to pass in the `comm_port` value.

Reply:
Gotcha! Added the comment to line 94.

Reply:
Thanks for making the change there, but none of the robots here use that xacro which seems a little weird.